tools/cegr_statistics/stats_util.py

```diff
@@ -142,10 +142,17 @@ def get_datasets(config_file, ids, datatypes):
     for i, t in zip(listify(ids), listify(datatypes)):
         d['id'] = i
         d['type'] = t
-        d['uri'] = '%s/datasets/%s/display?preview=True' % (defaults['GALAXY_BASE_URL'], i)
+        d['uri'] = '%s/datasets/%s/display?preview=False' % (defaults['GALAXY_BASE_URL'], i)
     return d
 
 
+# This function is written based on history url format in Galaxy 19.05 and it is server agnostic.
+def get_history_url(config_file, historyId):
+    # http://hermes.vmhost.psu.edu:8080/histories/view?id=1e8ab44153008be8
```

Comment:
Please remove this commented line.

Reply:
I have tried to follow Gregg template for "def get_datasets" function [(line 139) ](https://github.com/seqcode/cegr-galaxy/blob/master/tools/cegr_statistics/stats_util.py#L139)which he includes a sample of URL. But if you feel it is not good. I'll remove it.

Reply:
Could you please indicate that this a sample URL in the comment line?

Reply:
I implemented your comment. Please check it [here](https://github.com/seqcode/cegr-galaxy/pull/179/commits/e3eefe2451450d4987681c078fb31296500225d1#diff-c67ec22872d09c445b8ee1e22dab96d4L151)

Reply:
Thanks for the update.